### ISSUE TITLE
Sourcegraph.com: no code intel on kubernetes/kubernetes

### ISSUE THREAD

**Issue by dadlerj** [sourcegraph/sourcegraph#2773]
- **Sourcegraph version:** Sourcegraph.com
- **Platform information:** Mac, Chrome

#### Steps to reproduce:

1. Visit https://sourcegraph.com/github.com/kubernetes/kubernetes/-/blob/cmd/cloud-controller-manager/controller-manager.go#L41:17
3. Try to hover on a symbol

#### Expected behavior:

Code intelligence

#### Actual behavior:

No hover appears, error in console:

`
Error: unable to determine workspace's root Go import path: failed to fetch/write/open zip archive from https://REDACTED@sourcegraph.com/github.com/kubernetes/kubernetes@4b1282d9258e13791970a20395df4a6ec2613856/-/raw: failed to fetch missing archive cache item: zip URL https://REDACTED@sourcegraph.com/github.com/kubernetes/kubernetes@4b1282d9258e13791970a20395df4a6ec2613856/-/raw returned HTTP 500 (original rootPath is "git://github.com/kubernetes/kubernetes?4b1282d9258e13791970a20395df4a6ec2613856")
`

**Comment by chrismwendt** [comment]
I'm able to repro at https://sourcegraph.com/github.com/kubernetes/kubernetes/-/blob/cmd/cloud-controller-manager/controller-manager.go#L41:17 Investigating now.

**Comment by chrismwendt** [comment]
- HTTP 200 https://sourcegraph.com/github.com/gorilla/mux/-/raw
- HTTP 500 https://sourcegraph.com/github.com/kubernetes/kubernetes/-/raw

HTTP 500 trace https://app.lightstep.com/sourcegraph-prod/trace?span_guid=26b850dc0374d313&at_micros=1552684486060328#span-26b850dc0374d313

> failed to fetch missing archive cache item: failed to copy and close missing archive cache item: exit status 128 (stderr: "error: object file .git/objects/74/b1201277c0a87e8b2d9cc4747d62c691168c18 is empty error: object fil... (truncated)")

@slimsag Do you have any insight into this, or can you recommend troubleshooting steps?

**Comment by chrismwendt** [comment]
I found the empty file on gitserver-0 at `/data/repos/github.com/kubernetes/kubernetes/.git/objects/74/b1201277c0a87e8b2d9cc4747d62c691168c18`.

https://stackoverflow.com/a/12371337/2061958 has steps for fixing this, but it involves surgery on the .git directory. Since Sourcegraph.com is a mirror, I'm going to try deleting the whole repository and enabling again from the UI.

**Comment by chrismwendt** [comment]
I deleted kubernetes/kuberenetes, but now I can't figure out how to bring it back. Slack discussion: https://sourcegraph.slack.com/archives/C08JA8Q1H/p1552685623112000

I merged https://github.com/sourcegraph/deploy-sourcegraph-dot-com/pull/400, but that didn't fix it.

@keegancsmith I'll optimistically hand this off to you.

**Comment by keegancsmith** [comment]
We are investigating why you can't add k8s in https://github.com/sourcegraph/sourcegraph/issues/2933

**Comment by KattMingMing** [comment]
Another user reported that this is still an issue. Any information about what's going on?

**Comment by ankitpateledu** [comment]
still waiting for the fix

**Comment by keegancsmith** [comment]
I'll take a look at this tomorrow.

**Comment by nicksnyder** [comment]
Let's prioritize getting kubernetes/kubernetes working manually, even if there is a more general problem that needs to be solved.

**Comment by keegancsmith** [comment]
Sorry for the slow turn-around time. kubernetes repository is working again on sourcegraph.com. It seems manually deleting the repository sourcegraph.com couldn't recover from. That will be investigated in https://github.com/sourcegraph/sourcegraph/issues/2933 Note: This is a situation unique to sourcegraph.com, it does not affect our customer instances.

Manual steps: I ran a manual update on the production database. Additionally I confirmed there were no other soft deleted repositories.

```sql
update repo set deleted_at = null where name = 'github.com/kubernetes/kubernetes';
select count(1) from repo where deleted_at != null;
```
